src/style.css

```diff
@@ -0,0 +1,99 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
Use `*` only for set box-sizing. If u need to reset some default browser styles reset it manually for need tags

```
* {
  box-sizing: border-box;
}

p,
h1,
h2 {
  margin: 0;
}
```